server/sonar-server/src/main/java/org/sonar/server/ui/ws/SettingsAction.java

```diff
@@ -58,13 +58,13 @@ public void define(NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    boolean isRoot = userSession.isRoot();
+    boolean isSysAdmin = userSession.isSystemAdministrator();
```

Comment:
a UT should be added, right?
now System admin of default org can see when organizations are disabled